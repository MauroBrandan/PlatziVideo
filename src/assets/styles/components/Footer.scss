.footer {
	width: 100%;
	height: 100px;
	display: flex;
	align-items: center;
	background-color: var(--primary-color);
}

.footer__links {
	width: 24%;
	min-width: 420px;
	display: flex;
	justify-content: space-between;

	& a {
		color: white;
		cursor: pointer;
		font-size: 0.9rem;
		text-decoration: none;
	}

	& a:hover {
		text-decoration: underline;
	}
}
