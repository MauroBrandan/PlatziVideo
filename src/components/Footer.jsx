import React from 'react'
import '../assets/styles/components/Footer.scss'

const Footer = () => (
	<footer className='footer'>
		<div className='footer__links'>
			<a href='/'>Terminos de uso</a>
			<a href='/'>Declaraci√≥n de privacidad</a>
			<a href='/'>Centro de ayuda</a>
		</div>
	</footer>
)

export default Footer
