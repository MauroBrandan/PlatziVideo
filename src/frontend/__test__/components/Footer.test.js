import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'
import Footer from '../../components/Footer'

describe('<Footer />', () => {
	const footer = mount(
		<MemoryRouter>
			<Footer />
		</MemoryRouter>
	)

	test('Render Footer Component ', () => {
		expect(footer.length).toEqual(1)
	})

	test('Footer haves 3 anchors ', () => {
		expect(footer.find('Link')).toHaveLength(3)
	})

	test('Footer Snapshot', () => {
		const footer = create(
			<MemoryRouter>
				<Footer />
			</MemoryRouter>
		)
		expect(footer.toJSON()).toMatchSnapshot()
	})
})
