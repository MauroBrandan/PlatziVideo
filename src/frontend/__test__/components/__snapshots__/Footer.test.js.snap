// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer Snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Terminos de uso
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      Declaraci√≥n de privacidad
    </a>
    <a
      href="/"
      onClick={[Function]}
    >
      Centro de ayuda
    </a>
  </div>
</footer>
`;
